### DISCUSSION TITLE
Issue while creating an onnx runtime session

### DISCUSSION THREAD

**Discussion by shinron4** [microsoft/onnxruntime#15623] [General]
I am trying to create an onnx runtime session with a model, this mode has symbolic links to some custom operations. While creating the onnx runtime session I am getting the following trace.

  File "gen_onnx.py", line 205, in <module>
    main()
  File "gen_onnx.py", line 192, in main
    sess = rt.InferenceSession("Cylinder3d.onnx",so)
  File "/local/mnt/workspace/gaursing/Python_ENVS/Cylinder3D_New/lib/python3.8/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 360, in __init__
    self._create_inference_session(providers, provider_options, disabled_optimizers)
  File "/local/mnt/workspace/gaursing/Python_ENVS/Cylinder3D_New/lib/python3.8/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 397, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from Cylinder3d.onnx failed:Unsuported type proto value case.

My whole confusion is whether it is something related to the custom operation implementation not registered with the onnx_runtime or is it some issue with the model itself. But I am very sure the issue is later. because if the custom operation is not registered then we get following issue like 

onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from centerpoint_voxelnet_experiment_new_test.onnx failed:Fatal error: spconv:SparseConv(-1) is not a registered function/op

Please suggest what could be the problem with here.

**Comment by tianleiwu** [comment]
@shinron4, Try [onnx checker](https://onnx.ai/onnx/api/checker.html) if Cylinder3d.onnx does not contain custom ops.
